runtime/src/weights/extrinsic_weights.rs

```diff
@@ -24,7 +24,7 @@ pub mod constants {
 	parameter_types! {
 		/// Executing a NO-OP `System::remarks` Extrinsic.
 		pub const ExtrinsicBaseWeight: Weight =
-			Weight::from_ref_time(constants::WEIGHT_REF_TIME_PER_NANOS.saturating_mul(125_000));
+			Weight::from_parts(constants::WEIGHT_REF_TIME_PER_NANOS.saturating_mul(125_000), 0);
 	}
```

Comment:
Same as above

Reply:
this is what latest version of template is using: [here](https://github.com/substrate-developer-hub/substrate-parachain-template/blob/ebcc7250a829e5374f3aa19834b840c460741fd8/runtime/src/weights/extrinsic_weights.rs#L27)